taichi/transforms/auto_diff.cpp

```diff
@@ -593,17 +693,12 @@ class MakeAdjoint : public IRVisitor {
     }
   }
 
-  Stmt *insert_back(std::unique_ptr<Stmt> &&stmt) {
+  Stmt *insert_ad_transform_stmt(std::unique_ptr<Stmt> &&stmt) override {
```

Comment:
nit: shall we keep the old name `insert_back` for this function for now? ;) I guess you changed it likely because forward mode insert in a different place? That could be passed in as an additional arg or simply `insert_grad_stmt`?

Reply:
Have changed them to `insert_grad_stmt`.